go/tasks/plugins/k8s/kfoperators/pytorch/pytorch.go

```diff
@@ -0,0 +1,225 @@
+package pytorch
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/lyft/flyteidl/gen/pb-go/flyteidl/plugins"
+	flyteerr "github.com/lyft/flyteplugins/go/tasks/errors"
+	"github.com/lyft/flyteplugins/go/tasks/pluginmachinery"
+	"github.com/lyft/flyteplugins/go/tasks/pluginmachinery/flytek8s"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sort"
+	"time"
+
+	pluginsCore "github.com/lyft/flyteplugins/go/tasks/pluginmachinery/core"
+	"github.com/lyft/flyteplugins/go/tasks/pluginmachinery/k8s"
+	"github.com/lyft/flyteplugins/go/tasks/pluginmachinery/utils"
+
+	"github.com/lyft/flyteidl/gen/pb-go/flyteidl/core"
+	"github.com/lyft/flyteplugins/go/tasks/logs"
+	logUtils "github.com/lyft/flyteidl/clients/go/coreutils/logs"
+
+	//commonOp "github.com/kubeflow/common/pkg/apis/common/v1" // switch to real 'common' once https://github.com/kubeflow/pytorch-operator/issues/263 resolved
```

Comment:
interesting call out. I think after this for all kubeflow operators, we can just have a `flyte-kubeflow-commons` where all status checking can be done inline